Core/src/org/sleuthkit/autopsy/filequery/FileSearchData.java

```diff
@@ -172,45 +170,45 @@ static FileSize fromSize(long size) {
                 return UNDER_100KB;
             }
         }
-        
+
         /**
          * Get the upper limit of the range.
-         * 
+         *
          * @return the maximum file size that will fit in this range.
          */
         long getMaxBytes() {
             return maxBytes;
         }
-        
+
         /**
          * Get the lower limit of the range.
-         * 
+         *
          * @return the maximum file size that is not part of this range
          */
         long getMinBytes() {
             return minBytes;
         }
-        
+
         /**
          * Get the rank for sorting.
-         * 
+         *
          * @return the rank (lower should be displayed first)
          */
         int getRanking() {
             return ranking;
         }
-        
+
         @Override
         public String toString() {
             return displayName;
         }
     }
-   
+
     /**
-     * Enum representing the file type.
-     * We don't simply use FileTypeUtils.FileTypeCategory because: 
-     *   - Some file types categories overlap
-     *   - It is convenient to have the "OTHER" option for files that don't match the given types
+     * Enum representing the file type. We don't simply use
+     * FileTypeUtils.FileTypeCategory because: - Some file types categories
+     * overlap - It is convenient to have the "OTHER" option for files that
+     * don't match the given types
```

Comment:
Can you fix auto-format's mangling of this comment block?

Reply:
Yeah I will fix it until the next auto format I guess.